src/darwin/Framework/CHIP/MTRDeviceController.mm

```diff
@@ -148,11 +148,14 @@ - (os_unfair_lock_t)deviceMapLock
     return &_underlyingDeviceMapLock;
 }
 
-- (instancetype)initForSubclasses:(BOOL)startSuspended
+- (instancetype)initForSubclasses:(BOOL)startSuspended uniqueIdentifier:(NSUUID *)uniqueIdentifier
```

Comment:
General comment here to hold this until I test this, this uniqueIdentifier stuff was broken by a similar change last week, and I need to verify this against cleints.

Reply:
Yep, can do.  Let me know.